doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -454,6 +454,37 @@ Field              | Type       | Meaning
 `cpu_nano_cores`   | [`bigint`] | Approximate CPU usage, in billionths of a vCPU core.
 `memory_bytes`     | [`bigint`] | Approximate RAM usage, in bytes.
 
+### `mz_storage_source_statistics`
+
+The `mz_storage_source_statistics` table gives the last known storage-specific statistics
+for all processes of all extant storage replicas.
```

Comment:
I think we said we didn't want to use the term "replica" for these. For now, it seems we can stick to "storage hosts" for this.

Reply:
Or just use sth like "extant sources and sinks"

Reply:
I copied this from `mz_storage_host_metrics` above haha...will change them both